data/sql/updates/pending_db_world/rev_1643656464415474000.sql

```diff
@@ -0,0 +1,82 @@
+(84168, 13996, 469, 2677, 2677, 1, -7408.65, -917.047, 465.067, 15260, 64),
+(84169, 13996, 469, 2677, 2677, 1, -7484.71, -886.083, 465.067, 15260, 64),
+(84170, 13996, 469, 2677, 2677, 1, -7485.81, -883.083, 465.067, 15260, 64),
+(84171, 13996, 469, 2677, 2677, 1, -7472.84, -870.652, 465.067, 15260, 64),
+(84172, 13996, 469, 2677, 2677, 1, -7411.15, -909.451, 465.55, 15260, 64),
+(84173, 13996, 469, 2677, 2677, 1, -7464.36, -862.938, 465.067, 15260, 64),
+(84174, 13996, 469, 2677, 2677, 1, -7419.28, -895.444, 465.067, 15260, 64),
+(84175, 13996, 469, 2677, 2677, 1, -7414.11, -897.39, 465.067, 15260, 64),
+(84176, 13996, 469, 2677, 2677, 1, -7453.61, -860.511, 465.067, 15260, 64),
+(84177, 13996, 469, 2677, 2677, 1, -7411.48, -901.349, 465.067, 15260, 64),
+(84178, 13996, 469, 2677, 2677, 1, -7452.37, -863.403, 465.067, 15260, 64),
+(84179, 13996, 469, 2677, 2677, 1, -7448.91, -868.297, 465.067, 15260, 64),
+(84180, 13996, 469, 2677, 2677, 1, -7401.1, -918.476, 465.067, 15260, 64),
+(84181, 13996, 469, 2677, 2677, 1, -7398.45, -930.862, 465.067, 15260, 64),
+(84182, 13996, 469, 2677, 2677, 1, -7523.28, -1060.82, 449.325, 15260, 64),
+(84183, 13996, 469, 2677, 2677, 1, -7504.52, -1043.61, 449.325, 15260, 64),
+(84184, 13996, 469, 2677, 2677, 1, -7521.06, -1058.83, 449.325, 15260, 64),
+(84185, 13996, 469, 2677, 2677, 1, -7517.85, -1056.92, 449.325, 15260, 64),
+(84186, 13996, 469, 2677, 2677, 1, -7551.43, -1040.67, 449.325, 15260, 64),
+(84187, 13996, 469, 2677, 2677, 1, -7514.51, -1054.34, 449.325, 15260, 64),
+(84188, 13996, 469, 2677, 2677, 1, -7550.22, -1038.97, 449.325, 15260, 64),
+(84189, 13996, 469, 2677, 2677, 1, -7513.05, -1051.5, 449.325, 15260, 64),
+(84190, 13996, 469, 2677, 2677, 1, -7546.59, -1013.29, 449.325, 15260, 64),
+(84191, 13996, 469, 2677, 2677, 1, -7510.62, -1049.56, 449.325, 15260, 64),
+(84192, 13996, 469, 2677, 2677, 1, -7550.8, -1014.48, 449.325, 15260, 64),
+(84193, 13996, 469, 2677, 2677, 1, -7508.79, -1046.54, 449.325, 15260, 64),
+(84194, 13996, 469, 2677, 2677, 1, -7547.72, -1014.95, 449.325, 15260, 64),
+(84195, 13996, 469, 2677, 2677, 1, -7496.73, -1040.82, 449.325, 15260, 64),
+(84196, 13996, 469, 2677, 2677, 1, -7491.52, -1037.38, 449.325, 15260, 64),
+(84197, 13996, 469, 2677, 2677, 1, -7492.67, -1003.49, 408.652, 15260, 64),
+(84298, 13996, 469, 2677, 2677, 1, -7471.64, -1005.85, 408.648, 15260, 64),
+(84299, 13996, 469, 2677, 2677, 1, -7478.53, -996.153, 408.646, 15260, 64),
+(84200, 13996, 469, 2677, 2677, 1, -7486.45, -1025.62, 408.641, 15260, 64),
+(84201, 13996, 469, 2677, 2677, 1, -7490.95, -1014.89, 408.643, 15260, 64),
+(84202, 13996, 469, 2677, 2677, 1, -7515.74, -960.099, 449.325, 15260, 64),
+(84203, 13996, 469, 2677, 2677, 1, -7518.07, -933.761, 457.659, 15260, 64);
+
+-- Emote crafting for Technicians
+UPDATE `creature_template_addon` SET `emote`=133 WHERE `entry`=13996;
+
+-- Clean-up
+DELETE FROM `linked_respawn` WHERE `guid` >= 84533 AND `guid` <= 85779;
```

Comment:
use BETWEEN instead

Reply:
Can't avoid the C mentality, keep forgetting these exist